kustomize/commands/edit/set/setimage.go

```diff
@@ -204,52 +207,37 @@ func replaceDigest(image types.Image, digest string) types.Image {
 }
 
 func parse(arg string) (types.Image, error) {
-	// matches if there is an image name to overwrite
-	// <image>=<new-image><:|@><new-tag>
```

Comment:
Could you remain comments demonstrate command formats like this line?

Reply:
I came up with a comment I think explains ok.

Reply:
@koba1t I wonder if you were able to review the fixes

Reply:
Sorry to delay review.

Reply:
> Now this function should work for all the options defined in the InvalidArgs message

That may be true, I think that is enough to show the format.